package.json

```diff
@@ -3,7 +3,7 @@
   "type": "module",
   "private": true,
   "engines": {
-    "node": "18.16"
+    "node": ">=18.16"
```

Comment:
ah oops this was left over, but gonna leave it. I've fixed the reason we were pinned to 18.16 in the build script so this is fine now